website/docs/tutorial/1b-setting-up-snowflake.md

```diff
@@ -3,14 +3,299 @@ title: Setting up with Snowflake
+11. You can skip the final step of inviting other users to your account and click Skip & Complete to finish the setup. You’ll be able to go back and add users at any point within your account. 
+
+12. Congrats! Your account is officially set up and ready to rock. 
 
 ## Start developing in dbt Cloud
+
+1. On the welcome page click "Start Developing", which will take you to Cloud IDE.
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/dbt_cloud_welcome_page.png" title="dbt Cloud - Welcome Page" />
+</p>
+
+2. After the Cloud IDE loads, click the "Initialize your project" button in the upper left hand corner of the screen to initialize and create the directory structure within your project.
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/dbt_cloud_initialize_your_project.png" title="dbt Cloud - Initialize Your Project" />
+</p>
+
+3. Next you’ll want to commit all the newly created folders and files to your main branch. Click "commit", enter a brief message, and then click "Commit" again in the pop up window.
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/dbt_cloud_click_commit.png" title="dbt Cloud - Click Commit" />
+</p>
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/dbt_cloud_write_commit_message.png" title="dbt Cloud - Write Commit Message" />
+</p>
+
+4. Next we’re going to make sure that you can query the data we loaded earlier from within the IDE. Enter the following code into a tab in the IDE and click preview to view the results:
+`select * from raw.jaffle_shop.customers`
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/dbt_cloud_preview_results.png" title="dbt Cloud - Preview Results" />
+</p>
+
+5. Alright, we’re ready to run our first dbt command. In the run bar at the bottom of the screen enter `dbt run` and click "Enter".
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/dbt_cloud_dbt_run.png" title="dbt Cloud - dbt run" />
+</p>
+
+When you expand the run bar you should see something similar to the below. This run should have created the two example models in your development schema in Snowflake. By clicking on each model you should see more details about each run in the drop down menu. Next, go into your trial Snowflake account and open up the `ANALYTICS` database. You should be able to see that your development schema was created by this run and within that schema both of the models that you just ran.
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/dbt_cloud_run_results.png" title="dbt Cloud - Run Results" />
+</p>
+
+Awesome!  You are all set to leverage the power of dbt Cloud and Snowflake together.  As a recap, we just completed the following:
+
+* Set up a Snowflake trial account
+
+* Loaded training data into your Snowflake account
+
+* Created a dbt Cloud account, either through Partner Connect or through the account flow
+
+* Connected dbt Cloud and Snowflake
+
+* Set up the dbt Cloud IDE, queried data, and did your first dbt run
```

Comment:
```suggestion
* Set up a Snowflake trial account
* Loaded training data into your Snowflake account
* Created a dbt Cloud account, either through Partner Connect or through the account flow
* Connected dbt Cloud and Snowflake
* Set up the dbt Cloud IDE, queried data, and did your first dbt run
```
suggestion: making the list a bit more concise for readability.

Reply:
👍